api/src/main/java/ai/djl/nn/norm/LayerNorm.java

```diff
@@ -178,7 +178,7 @@ public void loadMetadata(byte loadVersion, DataInputStream is)
     }
 
     /** The Builder to construct a {@link LayerNorm}. */
-    public static final class Builder {
+    public static class Builder {
```

Comment:
This change is not relevant to this PR. Please revert this class.

Reply:
It is reverted here. I put it into the separate PR https://github.com/deepjavalibrary/djl/pull/2309.